static/js/components/GeoJSONGlobePlot.jsx

```diff
@@ -46,7 +68,11 @@ const GeoJSONGlobePlot = ({ skymap, sources, galaxies, instrument }) => {
     // eslint-disable-next-line
     galaxies,
     // eslint-disable-next-line
-    instrument
+    instrument,
+    // eslint-disable-next-line
+    observations,
+    // eslint-disable-next-line
+    options
```

Comment:
Just curious, what does eslint not like about these inputs?

Reply:
Got rid of this.